entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -287,6 +287,156 @@ This in turn removes ambiguity and confusion around methods like `HasForeignKey`
 
 Change configuration of owned type relationships to use the new API surface as shown in the example above.
 
+## Dependent entities sharing the table with the principal are now optional
+
+[Tracking Issue #9005](https://github.com/aspnet/EntityFrameworkCore/issues/9005)
+
+This change will be introduced in EF Core 3.0-preview 4.
+
+**Old behavior**
+
+Consider the following model:
+```C#
+public class Order
+{
+    public int Id { get; set; }
+    public int CustomerId { get; set; }
+    public OrderDetails Details { get; set; }
+}
+
+public class OrderDetails
+{
+    public int Id { get; set; }
+    public string ShippingAddress { get; set; }
+}
+```
+Before EF Core 3.0, if `OrderDetails` is owned by `Order` or explicitly mapped to the same table then an `OrderDetails` instance was always required when adding a new `Order`.
+
+
+**New behavior**
+
+Starting with 3.0, EF Core will allow to add an `Order` without an `OrderDetails` and will map all of the `OrderDetails` properties except the primary key to nullable columns.
+When querying EF Core will set `OrderDetails` to `null` if any of its required properties doesn't have a value or if it has no required properties besides the primary key and all properties are `null`.
+
+**Mitigations**
+
+If your model has a table sharing dependent with all optional columns, but the navigation pointing to it should never be `null` then a required property should be added to it or at least one property should have a value.
```

Comment:
As a first mitigation I would mention "Change the application to handle cases when the navigation is `null`", because this is very likely easier to do than the other mitigation.